### DISCUSSION TITLE
How to intercept if someone posted a public note in mastodon?

### DISCUSSION THREAD

**Discussion by yorreraj** [mastodon/mastodon#17014] [Q&A]
Hello, I just made an application that interacts with mastodon using the activitypub protocol. But how can I know if someone has posted something without mentioning someone, that is to say public post address to all followers. Thanks to all who will help me
Note that I have already specified the shared inbox but it is never called after the publication of a note in mastodon

**Comment by Gargron** [comment]
To receive a user’s posts to your inbox you must follow that user.

**Reply by yorreraj** [reply]
I have already followed the author of the publication. But when he mentions me in the publication I received it in my inbox, but when he publishes without mentioning anyone, nothing happens on my side

**Reply by yorreraj** [reply]
I tested with a user in another instance of mastodon, and I followed my account in mastodon.social and when I publish there without mentioning the user of the other instance that is subscribed to me the publication is displayed in real time in the other instance.
So my question is how do I do the same?

**Reply by trwnh** [reply]
Is it possible that Mastodon expects a sharedInbox and is incorrectly expanding the as:public scope at time of delivery?

**Reply by nightpool** [reply]
@yorreraj If you go to the users profile, does it show your account as following the user? it sounds like you didn't complete the follow activity

**Reply by yorreraj** [reply]
@nightpool yes I saw my account in the list of followers

**Reply by nightpool** [reply]
Sorry, we can't really help you troubleshoot your own code. If everything you say is correct, then it should be working. So there's something wrong somewhere in your code that's preventing it from working.

**Reply by yorreraj** [reply]
But if someone posts something in mastodon without mentioning a user, will it be sent to all their followers' inbox?

**Reply by Gargron** [reply]
Yes

**Reply by yorreraj** [reply]
I'll see then because that's my problem, I don't get anything when someone posts unless they mention me. Thanks to all
